pkg/orchestrator/util.go

```diff
@@ -35,37 +36,25 @@ const (
 	K8sDaemonSet
 	// K8sStatefulSet represents a Kubernetes StatefulSet
 	K8sStatefulSet
+	// lastElement represents the lastElement of the enums iota
+	lastElement
```

Comment:
It feels not idiomatic although I can see the convenience to reduce the burden a little bit.

Reply:
I agree, but I couldn't find anything better that. 
Alternatively we could go with updating this with the last value e.g. `<- Statefulset`, but then we would need to update this value or make sure it is the last one 🤷‍♂️ 🤔

Reply:
Let me undo this change to reduce the number of changes in this PR.